crates/nix_rs/src/copy.rs

```diff
@@ -27,6 +27,8 @@ pub async fn nix_copy(
     }
     cmd.run_with(|cmd| {
         cmd.args(args);
+        cmd.stdout(Stdio::null());
```

Comment:
Is this necessary?

Reply:
Otherwise stdout will be inherited from the parent process

Reply:
>  stdout will be inherited from the parent process

Which means the stdout of the process goes to the stdout of the parent process. Isn't this expected behaviour?

Reply:
Yes, it is. My bad

Reply:
Compare `main` branch:

https://github.com/juspay/omnix/actions/runs/11222470055/job/31194977412

<img width="537" alt="image" src="https://github.com/user-attachments/assets/b52061b4-482e-4e31-a3a9-2f7c1fcb6aa8">

with this PR:

https://github.com/juspay/omnix/actions/runs/11222577781/job/31195313989?pr=298

<img width="465" alt="image" src="https://github.com/user-attachments/assets/4e6fa638-f083-4e58-8ea4-70b17e63d0e9">